pytensor/link/pytorch/linker.py

```diff
@@ -25,9 +27,21 @@ def conversion_func_register(*args, **kwargs):
             self.gen_functors.append((f"_{name}", functor))
             return functor
 
+        def constants_wrapper(x, **kwargs):
+            x = pytorch_typify(x)
+
+            @torch.compiler.assume_constant_result
+            def torch_assume_constant(arg=x):
```

Comment:
This is causing some strange behavior in the compiler where `outs` is a bunch of functions (the constants specifically) which doesn't make a lot of sense to me. I'm still investigating the cause.

Reply:
It's possible this approach has issues since we don't wrap things in a nn.Module https://github.com/pytorch/pytorch/issues/124858 (we shouldn't need to.)